src/ontology/lexmatch/split-mapping-set/mondo_exactmatch_icd11.foundation.tsv

```diff
@@ -31,7 +32,8 @@ MONDO:0000239	adiaspiromycosis	skos:exactMatch	icd11.foundation:1111587867	Pulmo
 MONDO:0000239	adiaspiromycosis	skos:exactMatch	icd11.foundation:139402453	Adiaspirosis	semapv:LexicalMatching	oaklib	0.8	oio:hasExactSynonym	oio:hasExactSynonym	haplosporangiosis	LEXMATCH
 MONDO:0000239	adiaspiromycosis	skos:exactMatch	icd11.foundation:139402453	Adiaspirosis	semapv:LexicalMatching	oaklib	0.8	oio:hasExactSynonym	rdfs:label	adiaspirosis	LEXMATCH
 MONDO:0000240	invasive aspergillosis	skos:exactMatch	icd11.foundation:1314810340	Invasive aspergillosis	semapv:LexicalMatching	oaklib	0.8497788951776651	rdfs:label	rdfs:label	invasive aspergillosis	LEXMATCH
-MONDO:0000241	Keshan disease	skos:exactMatch	icd11.foundation:1307765114	Keshan disease due to selenium deficiency	semapv:LexicalMatching	oaklib	0.8	rdfs:label	oio:hasExactSynonym	keshan disease	LEXMATCH
+MONDO:0000241	Keshan disease	skos:exactMatch	icd11.foundation:1307765114	Keshan disease due to selenium deficiency	semapv:LexicalMatching	oaklib	0.8	rdfs:label	oio:hasExactSynonym	keshan disorder	LEXMATCH
+MONDO:0000241	Keshan disease	skos:exactMatch	icd11.foundation:1307765114	Keshan disease due to selenium deficiency	semapv:LexicalMatching	oaklib	0.8	rdfs:label	oio:hasExactSynonym	keshan disorder	LEXMATCH
```

Comment:
Another duplication?

Reply:
I looked in the test build I did 25-Jun-2024 after `main` was merged into `develop` and this duplication (MONDO:0000241	Keshan disease	skos:exactMatch	icd11.foundation:1307765114	Keshan disease due to selenium deficiency	semapv:LexicalMatching	oaklib	0.8	rdfs:label	oio:hasExactSynonymkeshan disorder	LEXMATCH) exists in the build done after merging `main` into `develop`.

On `main`, this duplication does not exist, e.g. https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/lexmatch/split-mapping-set/mondo_exactmatch_icd11.foundation.tsv#L34

Reply:
Hmm, good investigation. That is very strange.